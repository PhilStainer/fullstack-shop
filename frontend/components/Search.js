import React from 'react'

import StyledSearch from '#root/components/styles/StyledSearch'

const Search = () => {
  return (
    <StyledSearch>
      <input type="text" placeholder="Search" />
    </StyledSearch>
  )
}

export default Search
