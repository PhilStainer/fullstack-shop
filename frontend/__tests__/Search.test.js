import {render} from '@testing-library/react'

import Search from '../components/Search'

test('renders', () => {
  const {asFragment} = render(<Search />)

  expect(asFragment()).toMatchSnapshot()
})
