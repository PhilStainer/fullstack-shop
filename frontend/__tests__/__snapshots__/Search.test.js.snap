// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="StyledSearch-m4yy8y-0 cytWFA"
  >
    <input
      placeholder="Search"
      type="text"
    />
  </div>
</DocumentFragment>
`;
